test/gov/service_apireg.sh

```diff
@@ -282,12 +301,10 @@ cat <<EOF >$KEY_TEST_DIR/svc_gps.json
     {"name":"HZN_LOCATION_ACCURACY_KM","label":"","type":"string","defaultValue":"0.5"},
     {"name":"HZN_USE_GPS","label":"","type":"string","defaultValue":"false"}
   ],
-  "deployment": {
+  "deployment":{
     "services":{
-      "gps":{
-        "image":"openhorizon/amd64_gps:2.0.3",
-        "privileged":true,
-        "devices":["/dev/bus/usb/001/001:/dev/bus/usb/001/001"]
+      "cpu":{
```

Comment:
You changed all the service's network names to cpu. I wanted you to leave those unchanged. These changes will make it harder to debug problems in the future.